### DISCUSSION TITLE
Connecting issues - Redmi 10 5G

### DISCUSSION THREAD

**Discussion by Gotsche43** [srsran/srsRAN_Project#581] [General Help]
Sorry for not including this problem in my other issue.

Anyways.

I have tried to connect my UE Redmi 10 5G on my private 5G network consisting of srsRAN and Open5gs. I think I have managed to do all the necessary prework on the UE, like activating 5G SA on *#*#726633#*#*, apn, subscriber identification, sim card change (both roaming mcc/mnc and non roaming) etc.
I can easily see the network on my UE, but if I try hitting the network it's immediatly spits out "Cannot connect to this network at the moment. Try again later."

Likewise I have monitored the waterfall from my gNB and it looks okay (I really like to get some comment on my pictures actually).
I have a gpsdo on sync.
I use b200mini.

I include some logs. Please comment if I provide you the wrong logs. I'm a beginner in this world, so any ideas to troubleshoot will be a big help. 

Regards
![waterfallnr2](https://github.com/srsran/srsRAN_Project/assets/145973502/6a2d077d-0759-48d5-997d-a9fc355cef6f)
![waterfallnr1](https://github.com/srsran/srsRAN_Project/assets/145973502/b64ff718-687d-4c2d-b2e3-daa03cab73b7)

[logs.zip](https://github.com/srsran/srsRAN_Project/files/15109679/logs.zip)

**Comment by pgawlowicz** [comment]
@Gotsche43 any update on this issue?

Could you try to set `clock=external` and `sync=default`?

What I told you last time about this config was wrong, but now the docs have been updated:

```
ru_sdr: 
  clock: default                        # Optional TEXT (default). Specify the RF device clock source. Supported: [default, internal, external, gpsdo].
  sync: default                         # Optional TEXT (default). Specify the RF device time synchronization source. Supported: [default, internal, external, gpsdo].
```

**Reply by Gotsche43** [reply]
Hi Pgawlowicz,

Thank you, I will try to do the changes and return:-)

Regards

**Reply by pgawlowicz** [reply]
Hi @Gotsche43, any update on this issue?

**Reply by Gotsche43** [reply]
Im ended up using srsue config file with another ip address, and know my network is fully functional, with Xiaomi Redmi Note 10 5G, and b210/b200mini as gNB.
I used sync as external, but it works with and without LeoBodnar GPS DO.

**Reply by pgawlowicz** [reply]
nice! thanks for the feedback!
